### PR TITLE
add folder generate_test

### PR DESCRIPTION
**Pull request by eternal-web**

### CODE REVIEW

File: archive/Thematic/2020/Mailchimped/generate_test/Udemy.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+I had an app for growing my English vocabulary. Usually, I used it after running while trying to sink in my bathtub. Better and more effective than just wasting time on Instagram, right? The same approach can help to [learn](https://hackernoon.com/learning-regex-basics-in-ruby-4qel3y5i) 
+other things too.
+ ![](../0007- ✈ xxx_files/1d50fd86-9160-412a-a241-41c0dbd88dcf.gif
+
+
+And online lectures are more than just video + text content. 
+[Interactive learning](https://hackernoon.com/my-experiments-and-how-to-start-with-machine-learning-fzh63yrk)  
+with well-thought courses is on the rise. Even having 
+[ tests online](https://hackernoon.com/flawed-education-school-system-does-not-translate-to-education-u0n33wvk)
```
Comment (atherdon) [inline archive/Thematic/2020/Mailchimped/generate_test/Udemy.md:13]: remove empty space before tests

**Review by atherdon — COMMENTED** [review]
please take a look at my comment

File: archive/Thematic/2020/Mailchimped/generate_test/Udemy.md (hunk 1 above)
Comment (eternal-web) [reply archive/Thematic/2020/Mailchimped/generate_test/Udemy.md:13]: at the beginning of lines where +

Comment (atherdon) [reply archive/Thematic/2020/Mailchimped/generate_test/Udemy.md:13]: i'm talking only about line number 13

**Comment by atherdon** [comment]
feel free to replace images with a new path @eternal-web 

- https://gitlab.com/hackernoon/creative/-/raw/master/newsletters/logos/thematic/2020/November/udemy-logo.png
- https://gitlab.com/hackernoon/creative/-/raw/master/newsletters/memes/2020/november/20.11/two.gif
- https://gitlab.com/hackernoon/creative/-/raw/master/newsletters/memes/2020/november/20.11/three.gif
- https://gitlab.com/hackernoon/creative/-/raw/master/newsletters/memes/2020/november/20.11/six.gif
- https://gitlab.com/hackernoon/creative/-/raw/master/newsletters/memes/2020/november/20.11/seven.gif
- https://gitlab.com/hackernoon/creative/-/raw/master/newsletters/memes/2020/november/20.11/one.gif
- https://gitlab.com/hackernoon/creative/-/raw/master/newsletters/memes/2020/november/20.11/four.gif
- https://gitlab.com/hackernoon/creative/-/raw/master/newsletters/memes/2020/november/20.11/five.jpg

**Pull request merged**